modules/sdk-coin-avaxp/src/lib/importTxBuilder.ts

```diff
@@ -0,0 +1,83 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { BuildTransactionError, NotSupported, TransactionType } from '@bitgo/sdk-core';
+import { TransactionBuilder } from './transactionBuilder';
+import { BaseTx, ImportTx, PlatformVMConstants, Tx, UnsignedTx } from 'avalanche/dist/apis/platformvm';
+import utils from './utils';
+import { BN, Buffer as BufferAvax } from 'avalanche';
+
+export class ImportTxBuilder extends TransactionBuilder {
```

Comment:
Most of the methods are dupes of the ones in the exportTxBuilder.

Can we have an `AtomicTransactionBuilder` that extends `TransactionBuilder` and it's extended by `exportTxBuilder` and `ImportTxBuilder`?